app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
```

Comment:
Regaring the gcc2 patch, there are 2 options:
**_a_**) try to create a patch to make it build on x86_gcc2;
**_b_**) change **`x86_gcc2`** to **`!x86_gcc2`** in ARCHITECTURES and SECONDARY_ARCHITECTURES.
I'll dig option **_a_** to see if the patch is not too big. But maybe it's not worth doing **_a_**.